docs/maintainers/releasing.md

```diff
@@ -255,3 +255,22 @@ Currently the NuGet packages published to [powershell-core feed][ps-core-feed] o
 Maintainers are working on including the assemblies built for non-Windows platforms.
 
 [ps-core-feed]: https://powershell.myget.org/gallery/powershell-core
+
+
+Homebrew
+========
+
+After the release, you can update homebrew formula.
+
+On macOS:
+1. Make sure that you have [homebrew cask](https://caskroom.github.io/).
+1. `brew update`
+1. `cd /usr/local/Homebrew/Library/Taps/caskroom/homebrew-cask/Casks`
+1. Edit `./powershell.rb`, reference [file history](https://github.com/vors/homebrew-cask/commits/master/Casks/powershell.rb) for the guidelines:
+1. Update `version`
```

Comment:
these are supposed to be sub-bullets I think

can't these changes be scripted?